src/component/Maps/maps.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./maps.scss";
+import GoogleMapReact from "google-map-react";
+
+import { Icon } from "@iconify/react";
+import locationIcon from "@iconify/icons-mdi/map-marker";
+
+const google_maps_key = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+
+const LocationPin = ({ text }) => (
+  <div className="LocationPin">
+    <Icon icon={locationIcon} className="LocationPin__icon" />
+    <p className="LocationPin__text">{text}</p>
+  </div>
+);
+
+//Map component that takes in a location and zoom level
```

Comment:
We don't really need a comment here. Let's try and avoid putting in comments in our code. I know we've been taught to comment our code, but if we have to comment our code or feel the need to comment our code, then there is definitely something that we need to change. That's why we make useful variable names & clean and readable code. By the looks of it, I can tell that `Map` component takes in a `location` and `zoomLevel` prop.